File: app/src/main/java/com/monk/retrofit/mvp/model/RetrofitModle.java
package com.monk.retrofit.mvp.model;


import io.reactivex.Observable;
import okhttp3.ResponseBody;

/**
 * @author monk
 * @date 2019-02-19
 */
public interface RetrofitModle {

    Observable<ResponseBody> getData(String unitGuid);

}

File: module_fragment/src/main/java/com/monk/modulefragment/retrofit/mvp/model/RetrofitModle.kt
package com.monk.modulefragment.retrofit.mvp.model

import io.reactivex.Observable
import okhttp3.ResponseBody

/**
 * @author monk
 * @date 2019-02-19
 */
interface RetrofitModle {
    fun getData(unitGuid: String?): Observable<ResponseBody?>
}